lib/smart_proxy_openscap/openscap_lib.rb

```diff
@@ -63,4 +64,10 @@ def self.fullpath(path = Proxy::OpenSCAP::Plugin.settings.contentdir)
     pathname = Pathname.new(path)
     pathname.relative? ? pathname.expand_path(Sinatra::Base.root).to_s : path
   end
+
+  def self.execute(*cmd)
```

Comment:
If this raises an exception, should it be
```suggestion
  def self.execute!(*cmd)
```